jsapp/js/utils.ts

```diff
@@ -24,7 +25,17 @@ alertify.defaults.notifier.closeButton = true;
 const cookies = new Cookies();
 
 export function notify(msg: string, atype = 'success') {
-  alertify.notify(msg, atype);
+  switch (atype) {
+    case 'empty':
+      toast(msg); // No icon
+      break;
+    case 'error':
+      toast.error(msg);
+      break;
+    // To avoid changing too much default remains 'success'
+    default:
+      toast.success(msg);
+  }
```

Comment:
Code that I didn't test -- it's more of a verbose way of clarifying what I mean above. 
```typescript
export function notify(msg: string, atype = 'success') {
  // To avoid changing too much, the default remains 'success' if unspecified.
  //   e.g. notify('yay!') // success

  switch (atype) {
  
    case 'success':
      toast.success(msg);
      break;

    case 'error':
      toast.error(msg);
      break;

    case 'warning':
      toast(msg, {icon: '⚠️'});
      break;

    case 'empty':
      toast(msg); // No icon
      break;
      
    // Defensively render empty if we're passed an unknown atype,
    // in case we missed something.
    //   e.g. notify('mystery!', '?') //
    default:
      toast(msg); // No icon
      break;
  }
}
```
This could be condensed a bit with fallthrough cases, but we might have an eslint rule against that.